### PR TITLE
Fix javadoc issues

### PR DESCRIPTION
**Pull request by fenik17**
Motivation:

Invalid javadoc in project

Modifications:

Fix it

Result:

Correct javadoc

### CODE REVIEW

File: codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
  */
 package io.netty.handler.codec.http;
 
+import io.netty.handler.codec.http.cookie.ClientCookieDecoder;
+
```
Comment (slandelle) [inline codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java]: IMHO, it's not a good thing to add an import for the sheer sake of javadoc while the actual code doesn't use it. I would rather use the fully qualified class name in javadoc then.

File: codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java [hunk 2]
```diff
@@ -15,6 +15,8 @@
  */
 package io.netty.handler.codec.http;
 
+import io.netty.handler.codec.http.cookie.ServerCookieDecoder;
+
```
Comment (slandelle) [inline codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java]: Same as above, I would use the fully qualified class name in javadoc instead.

File: codec/src/main/java/io/netty/handler/codec/protobuf/ProtobufVarint32LengthFieldPrepender.java [hunk 3]
```diff
@@ -15,16 +15,16 @@
  */
 package io.netty.handler.codec.protobuf;
 
-import java.io.IOException;
-
+import com.google.protobuf.CodedOutputStream;
+import com.google.protobuf.nano.CodedOutputByteBufferNano;
```
Comment (slandelle) [inline codec/src/main/java/io/netty/handler/codec/protobuf/ProtobufVarint32LengthFieldPrepender.java]: Idem

File: codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java (hunk 1 above)
Comment (fenik17) [inline codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java]: Explicit import protects against javadoc breakage at compile time if a class renamed negligently.
And a simple class name is more comfortable in javadoc..

Comment (slandelle) [inline codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java]: Don't modern IDEs update the javadoc when performing such refactoring?

Comment (normanmaurer) [inline codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java]: We prefer to import in netty. So all good

**Comment by normanmaurer** [comment]
Cherry-picked into 4.1 (0623c6c5334bf43299e835cfcf86bfda19e2d4ce) and 4.0 (1ff8c762cd5228aaec2da6d464b1beb5e7eee276)

**Comment by fenik17** [comment]
@normanmaurer 4.0 has unresolved conflicts in HttpHeaders.java

**Comment by normanmaurer** [comment]
@fenik17 ups... fixed by 5d28bd1110b03ceae4c21f878e04a108d8e903ef

**Pull request closed without merging**